src/classes/storage.ts

```diff
@@ -1,24 +1,82 @@
-import { StopGroup, StopCluster, StopGroupGpuid, StopClusterGpuid } from '../types';
+import {
+  GroundPlacesFile,
+  StopGroup,
+  StopCluster,
+  GroundPlacesArray,
+  StopGroupGpuid,
+  StopClusterGpuid,
+  Gpuid,
+  GroundPlaceType,
+} from '../types';
 
 /**
  * @description Manipulate GroundPlaces.
  */
 export class Storage {
+  public readonly groundPlacesArray: GroundPlacesArray;
+
+  constructor(groundPlacesFile: GroundPlacesFile) {
+    this.groundPlacesArray = this.readJSONFile(groundPlacesFile);
+  }
+
+  /**
+   * @description Parse the JSON File that have all the ground places.
+   * @param jsonFile The JSON File to parse.
+   * @returns {GroundPlacesArray}
+   */
+  public readJSONFile(groundPlacesFile: GroundPlacesFile): GroundPlacesArray {
+    const stringifyJSON: string = JSON.stringify(groundPlacesFile);
+    const parsedJSON: GroundPlacesFile = JSON.parse(stringifyJSON);
```

Comment:
this seems strange to stringify then parse. There must be a cleaner way to do this